### PR TITLE
feat: add an initial toolbar to the editor

### PR DESCRIPTION
**Pull request by NuclearRedeye**
Added an initial toolbar with some placeholder buttons and menus.

### CODE REVIEW

File: src/app/containers/manuscript/index.tsx [hunk 1]
```diff
@@ -1,33 +1,29 @@
...
+import { isManuscriptLoaded } from '../../selectors/manuscript.selectors';
 import './styles.scss';
-import {ManuscriptEditor} from "./manuscript-editor";
+import { ManuscriptToolbar } from './manuscript-toolbar';
+import { ManuscriptEditor } from './manuscript-editor';
 
-const renderBackdrop = () => (<Backdrop open={true}> <CircularProgress color="inherit" /> </Backdrop>);
+const renderBackdrop = () => (
+  <Backdrop open={true}>
+    {' '}
```
Comment (ilya-zhuravel) [inline src/app/containers/manuscript/index.tsx:12]: I think either linter or prettifier adds these but we need to make it stop :)

File: src/app/containers/manuscript/manuscript-toolbar.tsx [hunk 2]
```diff
@@ -0,0 +1,119 @@
...
+  const invokeRedo = useCallback(() => dispatch(manuscriptActions.redoAction()), [dispatch]);
+  const invokeBold = useCallback(() => dispatch(manuscriptActions.boldAction()), [dispatch]);
+  const invokeItalicize = useCallback(() => dispatch(manuscriptActions.italicizeAction()), [dispatch]);
+  const invokeLink = useCallback(() => dispatch(manuscriptActions.linkAction()), [dispatch]);
+
+  const renderContent = () => (
+    <AppBar position='sticky'>
+      <Toolbar className='manuscript-toolbar'>
+        <IconButton disabled={true}>
+          {' '}
```
Comment (ilya-zhuravel) [inline src/app/containers/manuscript/manuscript-toolbar.tsx:41]: any chance we could remove these?

File: src/app/containers/manuscript/styles.scss [hunk 3]
```diff
@@ -1,9 +1,22 @@
...
     padding: 20px 0;
   }
 }
 
+.manuscript-toolbar {
+  width: 100%;
+  min-height: 64px;
+  margin: 0px;
+  background-color: #fafafa;
+  box-shadow: 0 4px 5px 0 rgba(0, 0, 0, 0.14), 0 1px 10px 0 rgba(0, 0, 0, 0.12), 0 2px 4px -1px rgba(0, 0, 0, 0.2);
```
Comment (ilya-zhuravel) [inline src/app/containers/manuscript/styles.scss:15]: I can't find any docs for this syntax. Does it set shadow for 3 sides? or is it 3 overlapping shadows?

File: src/app/containers/manuscript/index.tsx (hunk 1 above)
Comment (NuclearRedeye) [reply src/app/containers/manuscript/index.tsx]: I don't think it's the linter adding them, nor prettier. I might've missed it whilst rebasing and accidentally restored it. I've removed it for now though...

**Pull request merged**